### DISCUSSION TITLE
Click event listener not working on Livewire v3.

### DISCUSSION THREAD

**Discussion by dalwar9195** [livewire/livewire#6685] [1. Help]
I have a livewire component  where I have shown all users. I want to delete specific user through clicking a button. But it's not working for me. My delete button is not responding. I have attached my component codes in the attachment. Can you help me about this issue?

Thanks...
![Screenshot_1](https://github.com/livewire/livewire/assets/35813995/2ee8921e-6296-4feb-a988-64b288673833)
![Screenshot_2](https://github.com/livewire/livewire/assets/35813995/bb5a1ff2-5158-4b39-8f7b-f0c8aad4f4d7)

**Comment by joshhanley** [comment]
@mddalwar do you have any browser console errors?

Also please don't paste screenshots of code in future, as it makes it harder to help, instead paste code in code blocks.

When you paste code into github, you need to surround it in triple back ticks and then specify the file type like php or blade.

For example
\```php
\\ PHP code here
\```

\```blade
\\ HTML and blade code here
\```

Thanks!

**Reply by dalwar9195** [reply]
There is none of any browser error here.

Thanks for your suggestions.
